### ISSUE TITLE
support inheritdoc

### ISSUE THREAD

**Issue by wmchuang** [domaindrivendev/Swashbuckle.AspNetCore#1799]
I have a lot of Dtos, annotation like, so I used it inheritdoc,like 
`  /// <inheritdoc cref="TypeEnum"/>
        public string Type { get; set; }`

In the generated XML file is 
` <member name="P:*****Type">
            <inheritdoc cref="T:******TypeEnum"/>
        </member>`

After that,I can't see comments on UI. So, Can this method be supported?

**Comment by redoz** [comment]
@wmchuang It's relatively easy to implement this yourself, I've used something similar to this in the past to make this work:
```csharp
var xmlDocList = new List<XDocument>();

// load your xml docs into xmlDocList here

var members = new Dictionary<string, XElement>();

// map identifier to element
foreach (var xmlDoc in xmlDocList)
{
    var memberElementList = xmlDoc.XPathSelectElements("/doc/members/member[@name and not(inheritdoc)]");
    foreach (var memberElement in memberElementList)
        members.Add(memberElement.Attribute("name").Value, memberElement);
}

// replace inheritdoc
foreach (var xmlDoc in xmlDocList)
{
    var memberElementList = xmlDoc.XPathSelectElements("/doc/members/member[inheritdoc[@cref]]");
    foreach (var memberElement in memberElementList)
    {
        var inheritdocElement = memberElement.Element("inheritdoc");

        var cref = inheritdocElement.Attribute("cref").Value;
        if (members.TryGetValue(cref, out var realDocMember))
            inheritdocElement.Parent.ReplaceNodes(realDocMember.Nodes());
    }

    options.IncludeXmlComments(() => new XPathDocument(xmlDoc.CreateReader()), includeControllerXmlComments);
}
```

**Comment by wmchuang** [comment]
> @wmchuang自己实现相对容易，过去我使用类似的方法来完成这项工作：
> 
> ```cs
> var  xmlDocList  =  新 列表 < XDocument >（）;
> 
> //在此处将xml文档加载到xmlDocList中
> 
> VAR  成员 =  新 词典 < 串，的XElement >（）;
> 
> //映射标识符元素
> 的foreach（ VAR  xmlDoc中 在 xmlDocList）
> {
>     var  memberElementList  =  xmlDoc。XPathSelectElements（“ / doc / members / member [@name and not（inheritdoc）] ”）;
>     foreach（memberElementList 中的var  memberElement）
>          成员。添加（memberElement。属性（“名称”）。价值，memberElement）;  
> }
> 
> //替换inheritdoc 
> 的foreach（ VAR  xmlDoc中 在 xmlDocList）
> {
>     var  memberElementList  =  xmlDoc。XPathSelectElements（“ / doc / members / member [inheritdoc [@cref]] ”）;
>     foreach（memberElementList 中的var  memberElement）  
>     {
>         VAR  inheritdocElement  =  memberElement。元素（“ Inheritdoc ”）；
> 
>         var  cref  =  InheritdocElement。属性（“ cref ”）。价值 ;
>         如果（会员。TryGetValue（CREF，出 VAR  realDocMember））
>              inheritdocElement。父母。ReplaceNodes（realDocMember。节点（））;
>     }
> 
>     选项。IncludeXmlComments（（）=>  新 的XPathDocument（xmlDoc中。CreateReader（）），includeControllerXmlComments）;
> }
> ```

Yes, you are right,thanks

**Comment by pyce-lb** [comment]
> 
> 
> @wmchuang It's relatively easy to implement this yourself...

Is there any reason **not to include** this out of the box though?

**Comment by janseris** [comment]
This is still not supported
https://github.com/dotnet/Scaffolding/issues/2137

**Comment by janseris** [comment]
@domaindrivendev Could you consider reopening the issue? The problem is not solved.

**Comment by Danielku15** [comment]
Also it has to be considered that `cref` is not mandatory. e.g:

```cs
public interface IUpdateable 
{
    /// <summary>
    /// Gets or sets when then resource was updated the last time. 
    /// </summary>
    DateTime LastUpdate { get; set; }
}

public class Blog : IUpdateable 
{
    // <inheritdoc />
    public DateTime LastUpdate { get; set; }
}
```

Resolving the correct documentation might be a bit tricky. I found this code in DocFx which already shows how tricky it can be to resolve the docs.  I added a comment to https://github.com/dotnet/roslyn/discussions/50192 which is unfortunately already a bit old. I think it would be great if the compiler can already output an XML file with all `inheritdoc` and `include` tags handled and copied.
